configs/Fedora/fedora.toml

```diff
@@ -328,6 +328,11 @@ Filters = [
     ' info-files-without-install-info-postin',
     ' info-files-without-install-info-postun ',
     ' postin-without-install-info ',
+
+# In Fedora dependencies are generated from upstream metadata, so python checks
+# are not needed here. https://github.com/rpm-software-management/rpmlint/issues/1171
+    'python-missing-require',
+    'python-leftover-require',
```

Comment:
This is still useful. Especially when packages have leftover manual requires.

Reply:
ok, so maybe it's good to do not filter at all or just filter the missing-require one?

Reply:
That is what I mean, yes. Sorry if it wasn't clear.

Reply:
https://github.com/rpm-software-management/rpmlint/pull/1180